src/main/java/seedu/address/model/job/JobSalary.java

```diff
@@ -48,11 +48,11 @@ public boolean equals(Object other) {
         }
 
         JobSalary otherSalary = (JobSalary) other;
-        return value == otherSalary.value;
+        return value.equals(otherSalary.value);
     }
 
     @Override
     public String toString() {
-        return "$" + value;
+        return "Monthly Salary: $" + value;
```

Comment:
If JobSalary is supposed to store monthly salary, perhaps you should reflect that in the class JavaDoc and the usage message as well.

Reply:
Changed!

Reply:
I was thinking of adding a note like "(monthly)" beside salary, didn't think you would change everything lol. 

Originally I was thinking of keeping salary general and some of the future features you could specify the time period of the salary.

But this works too, LGTM.